packages/runtime-vapor/src/dom/prop.ts

```diff
@@ -185,13 +185,16 @@ export function setText(el: Text & { $txt?: string }, value: string): void {
 }
 
 /**
- * Used by setDynamicProps only, so need to guard with `toDisplayString`
+ * Used by
+ * - setDynamicProps, need to guard with `toDisplayString`
+ * - v-text on dynamic component, value passed here is already converted
  */
 export function setElementText(
   el: Node & { $txt?: string },
```

Comment:
Would this type signature/treatment of `el` within this function still be accurate for the new usage with dynamic components? For example could `.nodes` on the dynamic component be one of the other `Block` variants besides `Node`?

Reply:
Makes sense. I tested the same behavior with vdom components and realized the consideration here was indeed incomplete. I've refactored the PR. Welcome to review again.